Sources/ArcGISToolkit/Documentation.docc/Resources/JobManager/JobManagerTutorialView.swift

```diff
@@ -0,0 +1,138 @@
+import ArcGIS
+import ArcGISToolkit
+import SwiftUI
+
+struct JobManagerTutorialView: View {
+    /// The job manager used by this view.
+    @ObservedObject var jobManager = JobManager.shared
+    /// The job that this view shows data for.
+    @State private var job: Job?
+    /// The job's error in case of failure.
+    @State private var error: Error?
+    /// A Boolean value indicating if we are currently adding an offline map job.
+    @State private var isAddingOfflineMapJob = false
+    /// The job's status.
+    @State private var status: Job.Status = .notStarted
+    
+    init() {
+        // Ask the job manager to schedule background status checks for every 30 seconds.
+        jobManager.preferredBackgroundStatusCheckSchedule = .regularInterval(interval: 30)
+    }
+    
+    var body: some View {
+        VStack {
+            if let job {
+                ProgressView(job.progress)
+                    .progressViewStyle(.linear)
+                    .padding()
+                Button {
+                    jobManager.jobs.removeAll()
+                    self.job = nil
+                } label: {
+                    Text("Start New Job")
+                }
+                .disabled(status == .started)
+                .opacity(status == .started ? 0.0 : 1.0)
+                .buttonStyle(.bordered)
+                .padding()
+                .task() {
+                    for await jobStatus in job.$status {
+                        status = jobStatus
+                        if status == .failed || status == .succeeded {
+                            notifyJobCompleted()
```

Comment:
I think potentially this might need to be a task that takes an id where the id is the job. Otherwise I'm not sure if it will get cancelled and restarted as we hope. something to test.

Reply:
Per our conversation, I tested the task and the task runs for every new job that gets run.